test/syncenginetestutils.h

```diff
@@ -1204,7 +1204,7 @@ class FakeFolder
         // A new folder will update the local file state database on first sync.
         // To have a state matching what users will encounter, we have to a sync
         // using an identical local/remote file tree first.
-        syncOnce();
+        Q_ASSERT(syncOnce());
```

Comment:
Actually, i think that since we build in release mode in some configuration this does not fire.

(We should probably replace all the Q_ASSERT in this file with ENFORCE or something)